static/js/publisher/release/components/revisionsListRow.js

```diff
@@ -123,25 +124,30 @@ const RevisionsListRow = props => {
             )}
         </td>
       )}
+      {progressiveBeingCancelled && (
+        <td>
+          <em>Cancel &amp; release previous revision</em>
```

Comment:
Maybe I read too much into it, but it sounds to me like it's the previous revision that is canceled and released.

Reply:
I've tried to make it fit on 1 line. Maybe just "Cancel"?